### ISSUE TITLE
Windows shell admin error

### ISSUE THREAD

**Issue by Inokenty90** [kingToolbox/WindTerm#48]
with admin privileges
![изображение](https://user-images.githubusercontent.com/3055660/104431081-4e836500-55a9-11eb-9fce-6d90ede4862e.png)
without
![изображение](https://user-images.githubusercontent.com/3055660/104431224-783c8c00-55a9-11eb-92b8-4b98944280e4.png)
Error from gsudo when interrupt program with Ctrl+c

**Comment by kingToolbox** [comment]
Exit Code 0xC000013A means that gsudo has been terminated by ctrl+c. From the picture, it should be the ctrl+c that caused the gsudo to exit. And I think gsudo did not handle ctrl+c correctly.

I just released a new version [2.0.0_Prerelease_1](https://github.com/kingToolbox/WindTerm/releases/tag/2.0.0), which also contains the updated gsudo v0.7.3, please download and see if the same problem still exists, thank you.

**Comment by Inokenty90** [comment]
same error
![изображение](https://user-images.githubusercontent.com/3055660/104568220-a08fbd80-5671-11eb-9363-3d3e4dc7f96a.png)

**Comment by Inokenty90** [comment]
https://github.com/gerardog/gsudo/issues/53 strange but in issue said that in 0.7.2 this error already fixed

**Comment by kingToolbox** [comment]
Yes, I have seen that issue, so one possibility is that it has not been fixed completely.

When I tried to reproduce your problem, I found only pressing ctrl+c at the beginning of the session may cause gsudo to be stopped. I guess it is because the powershell process has not started yet. Once I saw the command prompt, I can no longer stop it by pressing ctrl+c. I have tried several environments with such results. So maybe this question should be submitted to gsudo. I am afraid there is nothing I can do.

**Comment by Inokenty90** [comment]
Tried to reproduce gsudo error without WindTerm and failed. Tried to start powershell from cmd, cmd from powershell, start cmd from Task Manager.

**Comment by kingToolbox** [comment]
Thank you for your hard work. Although not reproducing the problem elsewhere does not mean that it is only related to WindTerm, but this month I will continue to analyze this problem. Whether it can be resolved or not, I will tell you.

**Comment by Inokenty90** [comment]
Repeated error on virtual machine with windows 10x64. Need to disable UAC and  login as built-in Administrator account.

**Comment by kingToolbox** [comment]
When I saw "repeated", I was optimistic. Such problems can usually be solved well. I will create a vm as you said and try it.

I read the introduction of gsudo, it should be allowed to run under such preset conditions, but I don't know what role gsudo can play in this way, because its purpose is to run commands with elevated permission.

**Comment by Error996** [comment]
Fix in gsudo v1.0.2 Latest

Fixed 'gsudo cache' auto-elevation on PowerShell when path has spaces.
Fixed Cache auto not working with sudo alias or msys (#98)
gsudo failing in PowerShell with ExecutionPolicy=Restricted. (#101)
Fixed Ctrl-C Handling on de-elevation (#103)

**Comment by gerardog** [comment]
Hi. Let me know if this issue persists. BTW, latest gsudo is v1.2 😊

**Comment by kingToolbox** [comment]
@Error996  @gerardog 
The `gsudo` currently used is `v1.0.2`, and will be upgraded to the `1.2.0` version soon. So this bug should be fixed, but I don't have an environment to test it.

**Comment by kingToolbox** [comment]
@Error996  @gerardog 
I am happy to inform you that [WindTerm_2.4.0_Prerelease_2](https://github.com/kingToolbox/WindTerm/releases/tag/2.4-prerelease) has been released and `gsudo 1.2.0` has been packaged. Please download and check it, thank you.
